.github/CODEOWNERS

```diff
@@ -20,3 +20,4 @@
 /licenses @chipkent @rcaudy
 /docker @devinrsmith @jcferretti @rcaudy
 /engine/function/ @chipkent @kosak @rcaudy
+/py @chipkent @jmao-denver
```

Comment:
Add me, too. I like to be "code owner of last resort" on all of these, just in case.